modules/storages/app/models/storages/project_storage.rb

```diff
@@ -51,6 +51,10 @@ class Storages::ProjectStorage < ApplicationRecord
 
   scope :automatic, -> { where(project_folder_mode: 'automatic') }
 
+  def automatic_management_possible?
+    storage.present? && storage.provider_type_nextcloud? && storage.automatically_managed?
```

Comment:
🟡 not very happy about the naming of `storage.automatically_managed?` sounds very off in this context, when checking if auto management is possible for a project storage.

but, this is part of database already, just for a bad gut feeling, I wouldn't trigger a new migration yet.

Reply:
Hmm, I'm not sure I see the disconnect in the name- seems quite fitting from my POV as a project folder can only be automatically managed, if the linked storage is automatically managed. Am I missing sth? 🤔 

FWIW, if indeed the name is not fitting, it's fairly reachable to introduce an alias for `storage.automatically_managed?` that would be more fitting for this context- we can do so without a database migration. Any alternate names that come to mind?

Reply:
resolved in live discussion. @akabiru will raise another short PR introducing an alias here to get a more understandable interface to the storage.